src/app/+dashboard/nodes/nodes.component.ts

```diff
@@ -31,10 +31,10 @@ export class DashboardNodesComponent implements OnInit {
       select(getNodesDataCleanData),
       distinctUntilChanged((a, b) => a.join('|') === b.join('|')),
       // Wait until the next animation frame is ready or 0.5s, first of both. It makes the changes smoother.
-      throttle(() => merge(
-        interval(500),
-        interval(0, animationFrameScheduler),
-      )),
+      // throttle(() => merge(
```

Comment:
was this by intention?

Reply:
Yes, It is intentional. But I want to keep it there to work on it